src/hotspot/share/opto/compile.cpp

```diff
@@ -4453,6 +4453,44 @@ void Compile::print_inlining_commit() {
   print_inlining_reset();
 }
 
+void Compile::print_inlining_append_late(CallGenerator* cg, InliningResult result, const char* msg) {
+  assert(print_inlining() || print_intrinsics(), "PrintInlining off?");
+  C->print_inlining_assert_ready();
+  C->print_inlining_move_to(cg);
+
+  stringStream* old = _print_inlining_list->at(_print_inlining_idx)->ss();
+
+  auto old_size = old->size();
+  bool found_line_break = false;
+  while (!found_line_break && old_size > 0) {
+    if (old->base()[old_size - 1] == '\n') {
+      found_line_break = true;
+    }
+    old_size--;
+  }
```

Comment:
Isn't the '\n' always at the end?  Why do we need to search for it?  If we do need to search for it, then we could use `strchr` or `strrchr`.